pkg/api/defaults.go

```diff
@@ -749,6 +749,11 @@ func (p *Properties) setAgentProfileDefaults(isUpgrade, isScale bool) {
 func (p *Properties) setWindowsProfileDefaults(isUpgrade, isScale bool) {
 	windowsProfile := p.WindowsProfile
 	if !isUpgrade && !isScale {
+		if windowsProfile.SSHEnabled == nil {
+			sshEnabled := DefaultWindowsSshEnabled
+			windowsProfile.SSHEnabled = &sshEnabled
```

Comment:
you can skip the extra temporary var and use `windowsProfile.SSHEnabled = to.BoolPtr(DefaultWindowsSshEnabled )`